specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1254,9 +1254,14 @@
         },
         "/subscriptions/{subscriptionId}/providers/Microsoft.Security/topology": {

```

Comment:
type names should be plural (topologies)

Reply:
This is a different case, since we return single topology.